pagopa/pn-frontend
# feat(PN-12808): update copy

## Short description
Updated copy in Recapiti page and related modal.

## List of changes proposed in this pull request
- updated common.json PG/PF
- updated recapiti.json PG/PF
- updated PecContactItem copy PG/PF

## How to test
1. Login as PF/PG
2. Go to Recapiti page and check copy is the same of [figma](https://www.figma.com/design/DlzvNQzkecy7L2ym40OTge/Domicilio-digitale?node-id=615-27023&t=LDLkXUCfX57RoZ2i-4)
